pytorch_toolkit/object_detection/tests/common/test_case.py

```diff
@@ -65,6 +80,17 @@ def skip_if_cpu_is_not_supported(self):
             if 'cpu' not in training_targets:
                 self.skipTest('CPU is not supported.')
 
+        @staticmethod
+        def skip_non_instantiated_template_if_its_allowed(template_folder):
```

Comment:
I think you can skip testing template if does not exist always, there is another test that tests existence of required files of instantiated template. Just put some warning to the log.

Reply:
ok